interfaces/console/src/app/console/sites/page.tsx

```diff
@@ -58,18 +58,14 @@ export default function Page() {
     siteName: '',
     siteId: '',
   });
-  const [sitesStatus, setSitesStatus] = useState<
-    Record<
-      string,
-      {
-        status: string;
-        batteryStatus: string;
-        signalStrength: string;
-      }
-    >
-  >({});
 
-  const { refetch: refetchSites, loading: sitesLoading } = useGetSitesQuery({
+  const subscriptionsInitialized = useRef<Record<string, boolean>>({});
```

Comment:
Sorry I'm not getting why we need useRef here. Can you please explain

Reply:
`useRef` is used here to efficiently manage and track subscription states for sites, ensuring subscriptions are initialized once per site and cleaned up appropriately when the network changes, all without affecting UI rendering performance.

Reply:
@Brackleycassinga We shouldn't have websocket connection for each site. Frontend will get exhaust if we did. We should have one websocket connection for sites page and handel all these complex logic on BFF.